3rdparty/snmalloc/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ add_custom_command(
   DEPENDS musl_includes libcxx_includes)
 
 add_enclave_library(
-  oesnmalloc
+  oesnmalloc_obj
   OBJECT
   allocator.cpp
   # List the following as sources of oesnmalloc in order to ensure that
```

Comment:
Should https://github.com/openenclave/openenclave/blob/2485e710fcf6a2ba15ea78290ae1cdc99e5236d2/3rdparty/snmalloc/CMakeLists.txt#L38
also needs to be modified?

Reply:
Good catch. Fixed.
Note: snmalloc is currently sgx only.